### DISCUSSION TITLE
What is the best way to make an OMP/built in theme use only the Terminal Colors instead of it's own custom ANSI/HTML color codes?

### DISCUSSION THREAD

**Discussion by Derpitron** [JanDeDobbeleer/oh-my-posh#2912] [Troubleshoot] [answered]
I like the [emodipt-extend](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/themes/emodipt-extend.omp.json) theme's prompt, but I want to use my own terminal's color codes for it, as I switch between the Nord and Gruvbox theme at my whim.

Is there an easy way to do this, or any way at all?

**Comment by JanDeDobbeleer** [comment, marked as answer]
You can use the [default 16 ANSI color names](https://ohmyposh.dev/docs/configuration/colors#standard-colors). They will be the same reference.

**Reply by Derpitron** [reply]
Looks like manual find-and-replace it is, then
